programs/protocol_event/src/state/grouping.rs

```diff
@@ -0,0 +1,34 @@
+use crate::state::type_size::{vec_size, CHAR_SIZE, PUB_KEY_SIZE, U16_SIZE};
+use anchor_lang::prelude::*;
+
+pub const MAX_CODE_LENGTH: usize = 8;
+pub const MAX_NAME_STRING_LENGTH: usize = 50;
+
+#[account]
+pub struct Category {
```

Comment:
Based on Pawel's suggestion earlier in standup, maybe a file per account would be nice. 
```
state/category.rs
state/event_group.rs
etc
```